README.md

```diff
@@ -457,6 +457,8 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [AngelList Weekly](https://angel.co/newsletters). The best of the startup world, curated weekly.
 - [Startup Curated](https://startup.curated.co). The startup community’s newsletter with the best content from and for founders. [Archive](https://startup.curated.co/issues).
 - [Unvalidated Ideas](https://unvalidatedideas.com). Fresh startup and SaaS ideas that you can take out and market validate every week.
+- [Generated.Company](https://generated.company). Unleash Your Entrepreneurial Potential with AI-Powered Company Ideas. 10 Company ideas every single week.
```

Comment:
is there an archive or at least a sample to see before adding?

Reply:
There isn't an archive, but you receive mails like this:

<img width="707" alt="image" src="https://user-images.githubusercontent.com/15849319/230902203-efb4fb53-50e3-4f14-b8bf-8be6029bda81.png">

I used to use Ideasai.net but this one is just shorter and straight to the point.

Reply:
Thanks.